### DISCUSSION TITLE
Would it support to extract class from doxygen and insert to md file?

### DISCUSSION THREAD

**Discussion by QINZHAOYU** [JakubAndrysek/MkDoxy#107] [General]
Are there any plans to provide a usage similar to mkdocstrings, which extracts documents of class objects and inserts them into md documents? This will provide more flexible support for document generation.

**Comment by JakubAndrysek** [comment]
Hi, this is one of the [plugin functionality](https://mkdoxy.kubaandrysek.cz/#:~:text=Code%20snippets%3A%20Generate%20code%20snippets%20in%20place%20of%20your%20standard%20Markdown%20documentation.). 
You can include part of the generated documentation directly in your MD file.

Documentation: https://mkdoxy.kubaandrysek.cz/snippets/
Example usage: https://mkdoxy.kubaandrysek.cz/snippets/classes/#__tabbed_1_2

Have I answered your question?

**Reply by QINZHAOYU** [reply]
Thank you very much
